controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -625,7 +621,14 @@ func (r *StorageClassClaimReconciler) reconcileCephBlockPool() error {
 			return fmt.Errorf("Could not find  device set named default in Storage cluster")
 		}
 		pgUnitSize := util.GetPGBaseUnitSize(deviceSet.Count)
-		r.cephBlockPool.Labels[controllers.StorageConsumerNameLabel] = r.storageConsumer.Name
+		if r.cephBlockPool.Labels == nil {
+			r.cephBlockPool.SetLabels(map[string]string{
+				controllers.StorageConsumerNameLabel: r.storageConsumer.Name,
+			})
+		} else {
+			r.cephBlockPool.Labels[controllers.StorageConsumerNameLabel] = r.storageConsumer.Name
+		}
```

Comment:
```suggestion
		labels := cephBlockPool.GetLabels();
		if labels == nil {
			cephBlockPool.SetLabels(map[string]string{})
		}
		labels[controllers.StorageConsumerNameLabel] = r.storageConsumer.Name
```

or even better, you can do what we do in the deployer, have a util function like this one:
https://github.com/red-hat-storage/ocs-osd-deployer/blob/main/utils/utils.go#L46-L54

then replace the entire section with 
```suggestion
		AddLabel(cephBlockPool, controllers.StorageConsumerNameLabel, r.storageConsumer.Name)
```
And start using it in all places we need to set a label (but don't do it in this PR :) )